pkg/services/sqlstore/permissions/dashboard.go

```diff
@@ -91,7 +91,7 @@ func NewAccessControlDashboardPermissionFilter(user *models.SignedInUser, permis
 	if queryType == searchstore.TypeAlertFolder {
 		folderActions = append(folderActions, accesscontrol.ActionAlertingRuleRead)
 		if needEdit {
-			folderActions = append(folderActions, accesscontrol.ActionAlertingRuleUpdate)
+			folderActions = append(folderActions, accesscontrol.ActionAlertingRuleCreate)
```

Comment:
Should we not support both Create and Update?

Reply:
1. Currently, logical operation `OR` is not supported in this area. 
2. This change is done to match the dashboard action required when user reads dashboards (see below)
3. Creating a new alert rule as well as moving alert rule between folders require create permission (and delete for the folder the alert is moved from).

Reply:
I created an issue to address this https://github.com/grafana/grafana/issues/47230